src/main/interface/cli.c

```diff
@@ -309,9 +313,16 @@ typedef enum {
     DUMP_ALL = (1 << 3),
     DO_DIFF = (1 << 4),
     SHOW_DEFAULTS = (1 << 5),
-    HIDE_UNUSED = (1 << 6)
+    HIDE_UNUSED = (1 << 6),
+    DUMP_OSD_PROFILE = (1 << 7)
```

Comment:
Add a comma at the end here for better maintainability.

Reply:
Done.